libs/hdf-converters/src/asff-mapper/asff-mapper.ts

```diff
@@ -37,29 +43,22 @@ export enum SpecialCasing {
 function whichSpecialCase(finding: Record<string, unknown>): SpecialCasing {
   const productArn = _.get(finding, 'ProductArn') as string;
   if (
-    productArn.match(
-      /^arn:[^:]+:securityhub:[^:]+:[^:]*:product\/aws\/firewall-manager$/
-    )
-  ) {
-    return SpecialCasing.FirewallManager;
-  } else if (
-    productArn.match(
-      /^arn:[^:]+:securityhub:[^:]+:[^:]*:product\/prowler\/prowler$/
-    )
+    _.get(finding, 'ProductName') === 'Default' &&
+    _.get(finding, 'GeneratorId') === 'cms.Chef Inspec'
```

Comment:
Also, it seems like it is written differently in different names, `cms.Chef Inspec`, `CMS Chef InSpec`, and `CMSInSpec` at least.

Reply:
CMSInSpec = the enum's key which we use to help out the type system more or less
CMS Chef InSpec = the enum's value - only used on definition more or less.  we try to have it be as complete a name as possible for the developer's benefit if we ever need to look that tool up again or something.
cms.Chef Inspec = the string that was placed in the generator id field of the asff file that we're using to identify what kind of file it is and then apply the special processing.